dev/tests/hu.elte.txtuml.xtxtuml.tests/src/hu/elte/txtuml/xtxtuml/tests/parser/XtxtUMLParserTestUtils.xtend

```diff
@@ -136,7 +136,7 @@ class XtxtUMLParserTestUtils {
 		assertTrue(member instanceof TUAttribute);
 		val attribute = member as TUAttribute;
 
-		assertEquals(visibility, attribute.prefix.visibility);
+		assertEquals(visibility, attribute.prefix.modifiers.visibility);
```

Comment:
Were the tests for the new elements skipped intentionally? I understand that our primary concern here is to check the viability of the concept of externals, but I'm not entirely comfortable with omitting tests.